.github/workflows/tests.yml

```diff
@@ -13,27 +13,18 @@ on:
 jobs: # Docs: <https://git.io/JvxXE>
   php:
     name: PHP ${{ matrix.php }}, ${{ matrix.setup }} setup, laravel ${{ matrix.laravel }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-stable
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
Thanks for review. I fixed the wrong tag.